src/main/java/org/mariadb/jdbc/internal/queryresults/CmdInformationMultiple.java

```diff
@@ -168,13 +169,13 @@ public int getCurrentStatNumber() {
 
     @Override
     public boolean moreResults() {
```

Comment:
FWIW, this could be simplified into just deferring directly to `isCurrentUpdateCount()` since that already handles the `null` check.